assets/install-scripts/install.sh

```diff
@@ -299,6 +299,9 @@ install_teleport() {
     # shellcheck source=/dev/null
     . $OS_RELEASE
   fi
+  # Some $ID_LIKE values include multiple distro names in an arbitrary order, so
+  # evaluate the first one.
+  ID_LIKE="$(echo "${ID_LIKE}" | awk '{print $1}')"
```

Comment:
Making use of bash parameter substitution removes the dependency on awk

```console
% ID_LIKE="ubuntu debian some-other"
% echo "${ID_LIKE%% *}" # Trim the suffix starting with space, matching the longest possible string
ubuntu
```

```suggestion
  ID_LIKE="${ID_LIKE% *}"
```